model_read_only/migrations/14.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+            SELECT ir_model_id, res_groups_id
+            FROM ir_model_group_readonly_rel_tmp;
+        """,
+    )
+    data = env.cr.fetchall()
+    grouped = {}
```

Comment:
Could you use `defaultdict` function from [collections](https://docs.python.org/3/library/collections.html#collections.defaultdict):
```py
grouped = defaultdict(list)
```